secrets/jwsSigningKey.key

```diff
@@ -0,0 +1,3 @@
+-----BEGIN PRIVATE KEY-----
+some valid signature
```

Comment:
Can we not put a dev/test key here?

I.e. what happens if JWS signage is enabled and this default non-valid key is used?

Reply:
I agree. I think we should use a self generated default key pair and also, put the public key in the sdk-scheme-adapter image. Update docs to make it imperative for users/implementers to change the defaults in their own installations. We could include an openssl command to generate a new keypair.